services/mgmt/apimanagement/src/package_2018_06_preview/models.rs

```diff
@@ -231,6 +231,11 @@ pub mod api_management_service_base_properties {
         External,
         Internal,
     }
+    impl Default for VirtualNetworkType {
```

Comment:
Does it matter if the `impl` is in a netsted `mod`? Should it be moved?

Reply:
This currently is 
``` rust
pub mod api_management_service_base_properties {
    use super::*;
    #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
    pub enum VirtualNetworkType {
        None,
        External,
        Internal,
    }
    impl Default for VirtualNetworkType {
        fn default() -> Self {
            Self::None
        }
    }
}
```

Should it be this instead?
``` rust
pub mod api_management_service_base_properties {
    use super::*;
    #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
    pub enum VirtualNetworkType {
        None,
        External,
        Internal,
    }
}

impl Default for api_management_service_base_properties::VirtualNetworkType {
        fn default() -> Self {
            Self::None
        }
}
```
I'm just wondering if the location of the impl matters.